e2e/tests/steps_init_test.go

```diff
@@ -32,6 +32,12 @@ const (
 	// validatorB maps to suffix ID 002 and receives in the Send test.
 	validatorB = "002"
 	chainId    = "0001"
+
+	servicerA = "001"
+	appA      = "000"
+	serviceA  = "0001"
+
+	relaychainEth = "RelayChainETH" // used to refer to Ethereum chain when retrieving relaychain settings
 )
 
 type rootSuite struct {
```

Comment:
```suggestion
// TODO(#874, olshansky): Populate the app & servicer keys with the full set
type rootSuite struct {
```

I'm already working on it in https://github.com/pokt-network/pocket/pull/874/files

![Screenshot 2023-07-03 at 7 29 31 PM](https://github.com/pokt-network/pocket/assets/1892194/1c2ca84a-7f3f-4c0b-8456-cebdabe35544)

Reply:
Thank you, updated the PR.